package.json

```diff
@@ -39,7 +39,7 @@
     "js-sha256": "^0.9.0",
     "multihashes": "^0.4.14",
     "muport-did-resolver": "^0.3.1",
-    "orbit-db": "^0.22.1",
+    "orbit-db": "3box/orbit-db#3box",
```

Comment:
Why 3box branch and not master?

Reply:
i think its easier to maintain a fork if we can have a branch reflect orbit, give a bit more flexibility. Since branching off 22 release, then may want to branch off 23 rebase commits there (or start building in both). Could in theory probs do on master, just would have do a lot remote refs when arranging commits needed

Reply:
I see, thanks for clarifying!